scripts/tests/scripts_regression_tests.py

```diff
@@ -248,8 +248,11 @@ def test_a_createnewcase(self):
             shutil.rmtree(testdir)
 
         cls._testdirs.append(testdir)
-
-        run_cmd_assert_result(self, "%s/create_newcase --case %s --compset X --res f19_g16 --output-root %s" % (SCRIPT_DIR, testdir, cls._testroot), from_dir=SCRIPT_DIR)
+        if CIME.utils.get_model() == "cesm":
+            pesfile = os.path.join("..","src","drivers","mct","cime_config","config_pes.xml")
+            run_cmd_assert_result(self, "%s/create_newcase --case %s --compset 2000_SATM_XLND_SICE_SOCN_XROF_XGLC_SWAV --user-compset --run-unsupported --pesfile %s --res f19_g16 --output-root %s" % (SCRIPT_DIR, testdir, pesfile, cls._testroot), from_dir=SCRIPT_DIR)
+        else:
+            run_cmd_assert_result(self, "%s/create_newcase --case %s --compset X --res f19_g16 --output-root %s" % (SCRIPT_DIR, testdir, cls._testroot), from_dir=SCRIPT_DIR)
```

Comment:
Thanks for adding a test of user compset! It seems like we still want to do the test without `--user-compset` for cesm too, right? i.e., take that second create_newcase command out of the else clause and just always do it?